Base/usr/share/man/man1/TextEditor.md

```diff
@@ -1,6 +1,6 @@
 ## Name
 
-TextEditor - SerenityOS text editor
+### ![Icon](/res/icons/16x16/app-text-editor.png) TextEditor - SerenityOS text editor
```

Comment:
Why does just this name have a header prefix (`###`)? 
Remove? 
```suggestion
![Icon](/res/icons/16x16/app-text-editor.png) TextEditor - SerenityOS text editor
```

Reply:
Whoops! Forgot to remove that one.